docs/logging.md

```diff
@@ -0,0 +1,160 @@
+---
+id: logging
+title: Logging
+---
+
+### Logging
+
+Almost every application nowadays does logging, whether it's logging of business events, history of some state of 
+application or even debug record sequences. We log to files, distributed systems, standard outputs of OSes, and we 
+got used to it so much it seems really simply to us. Many existing libraries provide many ways of logging from 
+your application's code, either automatic or manual.
+
+### What's wrong?
+
+First, in fact, logging is a side effect. Whether we log anything to a standard output, or a file - it's an interaction 
```

Comment:
Unclear as to whether "in fact" relates to the subsequent statement "logging is a side effect" or it is stylistically incorrect use of transitional phrases.
Assuming the former, I would recommend recomposing this sentence as "First, logging is, in fact, a side effect"